addons/sale_gelato/views/product_template_views.xml

```diff
@@ -37,7 +37,7 @@
                                 'no_create': True,
                                 'no_quick_create': True,
                                 'no_open': True,
```

Comment:
what is no_open? Might be dead code, and anyway it doesn't look compatible with open_form